sdk/src/wallet/core/mod.rs

```diff
@@ -230,3 +473,254 @@ impl<S: SecretManage> Drop for Wallet<S> {
         log::debug!("drop Wallet");
     }
 }
+
+/// Dto for the wallet data.
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct WalletDataDto {
+    /// The BIP44 path of the wallet.
```

Comment:
We can probably remove the doc comments for the Dto

Reply:
I think this has been addressed